packages/language-support/package.json

```diff
@@ -27,10 +27,10 @@
   "types": "./dist/esm/project/language-support/src/index.d.ts",
   "exports": {
     ".": {
+      "types": "./dist/esm/project/language-support/src/index.d.ts",
```

Comment:
apparently the order matters.. So types should go first and default last (we had warning in console on build about this)